FundacionParaguaya/MentorApp
# Bugfix/1137/add achievements to pdf

**Problem**: Achievement table was not created in the generated pdf for the Lifemap when downloaded. 

**To test:** 

1. Create Lifemap. Add few achievements.  At the Final screen click on download PDF button. Check if inside the generated pdf, separate page with Achievements was added.
